src/lib.rs

```diff
@@ -46,6 +46,10 @@ pub use tests::ffi::*;
 /// }
 /// ```
 ///
+/// If you require more control over the rustls `ClientConfig`, you can
```

Comment:
Nit: Maybe leave an extra comment at the top of the documentation block that mentions it uses safe defaults so people have an example of why they might need tighter control.

Reply:
Added some text to emphasize that `tls_config` uses safe defaults.